src/blog/components/informationMessage/InformationMessage.tsx

```diff
@@ -0,0 +1,67 @@
+import Button from "src/components/buttons/Button";
+import LinkButton from "src/components/linkButton/LinkButton";
+import Text from "src/components/text/Text";
+import { ILink, LinkType } from "studio/lib/payloads/navigation";
+import styles from "./informationMessage.module.css";
+import React from "react";
+
+export const homeLink = {
+  _key: "return-home",
+  _type: "link",
+  linkTitle: "Return to home",
+  linkType: LinkType.Internal,
+  internalLink: {
+    _ref: "/",
+  },
+};
+
+export const studioLink = {
+  _key: "go-to-studio",
+  _type: "link",
+  linkTitle: "Go to studio",
+  linkType: LinkType.Internal,
+  internalLink: { _ref: "/studio" },
+};
+
+interface InformationMessageProps {
+  title: string;
+  body: string;
+  button?: {
```

Comment:
I think we can remove button in the InformationMessage for now — I don't think we'd need a function related, just yet. If we do we can add it when we get there